spec/active_record/events_spec.rb

```diff
@@ -41,6 +41,19 @@
     it 'defines an inverse scope' do
       expect(Task.not_completed).to include(task)
     end
+
+    it 'allows overriding methods defined by the gem' do
+      task.class.class_eval do
+        def complete
+          super
+        end
+      end
+
+      task.complete
```

Comment:
Let's use `complete!`, which is more fundamental - `complete` has an additional condition.